docs/parameters.yaml

```diff
@@ -1786,10 +1786,10 @@ components: ["registry"]
 ############################
 #   Server-level configs   #
 ############################
-name: Server.TLSCertificate
+name: Server.TLSCertificateChain
```

Comment:
Instead of renaming this, please create a separate parameter and deprecate this one while labeling Server.TLSCertificateChain as the replacement. Then ensure that our deprecated assignment function in the config does the reassignment properly.